pkg/client/gen_test.go

```diff
@@ -203,15 +203,15 @@ func TestGenerateManifestGolden(t *testing.T) {
 			},
 			goldenFile: filepath.Join("testdata", "ssh.golden"),
 		}, {
-			name: "Empty array leads to 0 plugins (legacy plugin choice)",
+			name: "Empty array leads to default plugins, not 0",
```

Comment:
This is the "breaking change" that I dont think matters.

I was the original proponent of the 'run 0 plugins' case but I've gotten over it and I dont think anyone ever actually ran it that way. It also makes it harder for users to know the difference between empty and nil.

This idea that 0/nil both have the same behavior is much more intuitive.